cmds/generate_timestamp.go

```diff
@@ -0,0 +1,34 @@
+package cmds
+
+import (
+	"io/ioutil"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var generateTimestampDoc = `
+Write the current UTC timestamp to the file 'testtrack/build_timestamp.txt'.
+This timestamp is used by TestTrack clients to request the version of the split
+registry active at the given time.
+`
+
+func init() {
+	rootCmd.AddCommand(generateTimestampCmd)
+}
+
+var generateTimestampCmd = &cobra.Command{
+	Use:   "generate_timestamp",
+	Short: "Write the current UTC timestamp to 'testtrack/build_timestamp.txt'",
+	Long:  generateTimestampDoc,
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return generateTimestamp()
+	},
+}
+
+func generateTimestamp() error {
+	timestamp := []byte(time.Now().UTC().Format("2006-01-02T15:04:05Z"))
+
+	return ioutil.WriteFile("testtrack/build_timestamp.txt", timestamp, 0644)
```

Comment:
so here you're just letting RunE do it's magic and blow up if WriteFile errors? does the cli command need to return any output to the user at all?

Reply:
The most common user error that I can think of is if a user runs this command outside of a TestTrack project. In that case the testtrack directory won't exist.

Did some testing and the error it kicks out looks like:

```
Error: open testtrack/build_timestamp.txt: no such file or directory
Usage:
  testtrack generate_timestamp [flags]

Flags:
  -h, --help   help for generate_timestamp

exit status 1
```

Discussed earlier with @aburgel and think this error should be enough information. If some other error occurs, I believe the `WriteFile` error should be descriptive enough.

Reply:
well i mean in the successful case

Reply:
does the user need to know anything if it doesn't fail? if not cool.

Reply:
That's a good question. `generate_timestamp` should be getting called by a machine and not a human so I think just error code is enough.